config-user_EG_levante.yml

```diff
@@ -0,0 +1,82 @@
+###############################################################################
```

Comment:
I think this file should not be included in the PR, but I appreciate that it helps me test the recipe! :)